cupy/_core/dlpack.pxd

```diff
@@ -2,3 +2,8 @@ from cupy._core.core cimport ndarray
 
 cpdef object toDlpack(ndarray array) except +
 cpdef ndarray fromDlpack(object dltensor) except +
+cpdef from_dlpack(array)
+
+cdef enum:
+    device_CUDA = 2  # kDLCUDA
+    device_ROCM = 10  # kDLROCM
```

Comment:
```suggestion
+cdef extern from './include/cupy/dlpack/dlpack.h' nogil:
+    int device_CUDA 'kDLGPU'
+    int device_ROCM 'kDLROCM'
```

Reply:
I have no idea why I didn't do this! Thanks @kmaehashi 😅